### ISSUE TITLE
[Windows] - Search message sent and contact match

### ISSUE THREAD

**Issue by M4v3r1cK87** [kontalk/desktopclient-java#34]
Hello,
I trying the search message function, but I notice that when the search find the message, if is a sent message, is impossible to view what is the contact that I sent it.
![search](https://cloud.githubusercontent.com/assets/11871250/8673590/1c4798f6-2a38-11e5-80de-79b1f2d99ea0.PNG)

**Comment by daniele-athome** [comment]
Might be related to #33 if encryption blocks database processing someway?

**Comment by abika** [comment]
No, that is unrelated.

All three lists (Chats, contacts, current chat messages) are filtered at the same time. Yes, the current chat should always stay in the "Chats" list, never noticed that

**Comment by M4v3r1cK87** [comment]
Hi @abika 
I notice that if search somethings, the contacts list is blank and in the chat show the result searched. But if search a send message, is not showing the reciver because in the message popup you don't show the receiver (correct, search my message that I have sent):
![search send](https://cloud.githubusercontent.com/assets/11871250/8693709/5d37c2d6-2ad9-11e5-83cf-b092393c6bf4.PNG)

While, if I search a text that I received, in the message popup you show the receiver so the search showing the receiver :
![message received](https://cloud.githubusercontent.com/assets/11871250/8693766/dae9e506-2ad9-11e5-8b46-a3eec5adaaa8.PNG)

Another problem is if I search a text with a capital letter. In this example I searched the same previous word, but now the search is empty:
![search empty](https://cloud.githubusercontent.com/assets/11871250/8693805/3467e240-2ada-11e5-809d-f308e7cd5810.PNG)

And seems that the search not is for all contact but only for the contact selected. If I sent "hello" to 3 contacts, and after search the text "hello", I found the text only for the contact selected.
